Dshop/apps/products_catalogue/api_urls.py

```diff
@@ -1,11 +1,12 @@
 from django.urls import path, include
-from .api_views import ProductViewSet, CartAPIView
+from .api_views import OrderAPIView, ProductViewSet, CartAPIView
 from rest_framework.routers import DefaultRouter
 
 router = DefaultRouter()
 router.register('', ProductViewSet, basename='products-api')
```

Comment:
Just change this line a bit:
```suggestion
router.register('products/', ProductViewSet, basename='products-api')
```
And in global urls.py line 43 remove products from path. Thats it. 
Then you will have something like this

![image](https://github.com/pymasterspl/Dshop/assets/989256/b5c349b1-c009-4111-98c8-6d42455c5eaf)